buildroot/package/brutefir/brutefir.mk

```diff
@@ -0,0 +1,22 @@
+################################################################################
+#
+# brutefir
+#
+################################################################################
+
+BRUTEFIR_VERSION = 21f5a980371da9d3705cf87c2dc36252b343bfad
+BRUTEFIR_SITE = https://github.com/soundart/brutefir.git
```

Comment:
Why are you using a fork and not the official repository?

Reply:
Hi,

I added some features:
- cmake based build
- NEON optimisations (currently not active in this pull request)
- aarch64 timestamp

to get it running on ARM / openwrt and buildroot.

I contacted Anders Torger (in 2019 or 2020) wrt to these changes,  
but he was not interested(+had no time) at that time. I hope this will change later on.
In my repo the official release is on branch master, my changes are on the other branches.